magit-annex.el

```diff
@@ -388,14 +388,15 @@ With a prefix argument, prompt for FILE.
      ,(format "%s FILES.\n\n  git annex %s [ARGS] [FILE...]"
               (capitalize command) command)
      (interactive
-      (list (let ((atpoint (magit-annex-list-file-at-point)))
+      (list (let* ((selectedfiles (magit-annex-list-files-at-point-or-region))
```

Comment:
I think it'd be better to drop the `selectedfiles` variable since you only really need it to define the value for `default`.  So something like `(default (--when-let ...)`.